src/pages/bubblegum-v2/hashed-nft-data.md

```
@@ -148,14 +168,26 @@ pub enum LeafSchema {
+        data_hash: [u8; 32],
+        creator_hash: [u8; 32],
+        collection_hash: [u8; 32],
+        asset_data_hash: [u8; 32],
+        flags: u8,
+    },
 }
 ```
 
-Other than data and creator hashes, the leaf schema contains the following other items:
+Other than data, creator, collection and asset data hashes, the leaf schema contains the following other items:
 * nonce: This is a "number used once" value that is unique for each leaf on the tree.  It is needed to ensure Merkle tree leaves are unique.  In practice it retrieved from off-chain indexers, similar to asset proofs.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)